src/transport/assembly_dg.hh

```diff
@@ -130,8 +134,8 @@ public:
     /// Implements @p AssemblyBase::reallocate_cache.
     void reallocate_cache() override
```

Comment:
Can we move this method to GenericAssembly? Just name every particular assembly class or use `type(T).name()`
with particular assembly class `T`.